### PR TITLE
Create 2015-05-20-text-hover-bar

### PR DESCRIPTION
**Pull request by aneda**
I am not that happy with this one but I wanted to keep it simple so text-with-annotations would be easy to follow

### CODE REVIEW

File: _posts/python/bar/2015-05-20-text-hover-bar [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        x=['Product A', 'Product B', 'Product C'],
+        y=[20, 14, 23],
+        text = ['27% market share', '24% market share', '19% market share'],    
+        marker = Marker(
+                color = 'rgb(158,202,225)',
+                opacity = 0.6,
+                line =Line(
+                    color ='rgb(8,48,107)',
+                    width = 1.5
+                ),
```
Comment (chriddyp) [inline _posts/python/bar/2015-05-20-text-hover-bar]: remove this trailing comma

File: _posts/python/bar/2015-05-20-text-hover-bar [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        ),
```
Comment (chriddyp) [inline _posts/python/bar/2015-05-20-text-hover-bar]: and this one!

File: _posts/python/bar/2015-05-20-text-hover-bar [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    width = 1.5
+                ),
+        ),    
+)
+
+data = Data([trace0])
+
+layout = Layout( 
+
+  title= 'January 2013 Sales Report',
```
Comment (chriddyp) [inline _posts/python/bar/2015-05-20-text-hover-bar]: and keep the same indentation across the example - 4 spaces

File: _posts/python/bar/2015-05-20-text-hover-bar [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    
+     yaxis=YAxis(
+        showgrid = True, 
+        zeroline = True, 
+        showline = True,
+        tickfont = Font (
+                size = 16,
+                color = 'rgb(8, 48, 107)',
+        ),
+    ),
```
Comment (chriddyp) [inline _posts/python/bar/2015-05-20-text-hover-bar]: those colors look great! where does this example fall in the chapter? If it is *just* about adding hover text then we might want to remove the axis styling and introduce it later

File: _posts/python/bar/2015-05-20-text-hover-bar [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        showgrid = True, 
+        zeroline = True, 
+        showline = True,
+        tickfont = Font (
+                size = 16,
+                color = 'rgb(8, 48, 107)',
+        ),
+    ), 
+    
+    showlegend = False,
```
Comment (chriddyp) [inline _posts/python/bar/2015-05-20-text-hover-bar]: if there is only 1 trace, then `showlegend` shouldn't be necessary, it'll be hidden by default

**Comment by aneda** [comment]
@chriddyp is this good?

**Pull request merged**